src/WebJobs.Script/Config/ConfigurationSectionNames.cs

```diff
@@ -10,5 +10,6 @@ public static class ConfigurationSectionNames
         public const string Logging = "logging";
         public const string Aggregator = "aggregator";
         public const string HealthMonitor = "healthMonitor";
+        public const string HostIdPath = WebHost + ":hostid";
```

Comment:
I've been wondering about the convention here. All the other ones also expect to be joined with JobHost. Should we start doing a `ConfigurationPath.Combine(JobHost, Logging)` (for example) to return the full path here?

Reply:
Won't be able to do that with constants, but I'm in favor of having the path.